### PR TITLE
feat: add toPascalCase

### PR DESCRIPTION
**Pull request by imcvampire**
Issue: #16

### CODE REVIEW

**Comment by MoeSattler** [comment]
Hi @imcvampire, we'll have a look!

**Comment by MoeSattler** [comment]
@imcvampire can you rebase against `master`?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "case-converter",
   "description": "transform object to use different case conventions",
+  "version": "1.1.0",
```
Comment (MoeSattler) [inline package.json]: Don't bump the version manually please. semantic-release will take care of this.

Comment (imcvampire) [reply package.json]: Removed

**Comment by MoeSattler** [comment]
@imcvampire Merged!

Please not that it was released under [@travelperksl/case-converter](https://www.npmjs.com/package/@travelperksl/case-converter)

**Pull request merged**